RecoLocalCalo/HcalRecAlgos/interface/DoMahiAlgo.h

```diff
@@ -0,0 +1,146 @@
+#ifndef RecoLocalCalo_HcalRecAlgos_DoMahiAlgo_HH
+#define RecoLocalCalo_HcalRecAlgos_DoMahiAlgo_HH
```

Comment:
please drop "Do" from the name.
Perhaps "MahiFit" or similar is better because "Algo" corresponds to a different functionality in HCAL local reco.